HostLibraryTests/llvm/ArithmeticUnitPredicate_test.cpp

```diff
@@ -38,7 +38,8 @@ TEST(ArithmeticUnitPredicateTest, Any)
                         "value: [{type: TruePred}, \n"
                         "        {type: ArithmeticUnitCompatible, value: Any}]";
 
-    llvm::yaml::Input yin(mydoc);
+    LibraryIOContext<ContractionSolution>  context{std::string(""), {}, nullptr};
+    llvm::yaml::Input yin(mydoc, &context);
```

Comment:
The MasterSolutionLibrary serialization sets and needs the third parameter, if we don't have a pointer to the struct in there it will segfault. The context is usually set within the LoadLibraryFile, but since this is calling the serialization directly we need to do it here.